packages/appstore/src/types/common.types.ts

```diff
@@ -183,3 +183,17 @@ export type TLinkedTo = {
     balance?: string;
     currency?: string;
 };
+
+export type TWalletAccount = {
+    account_category?: 'trading' | 'wallets';
+    account_type?: string;
+    balance: string | number;
+    currency: string;
+    is_disabled: boolean;
+    is_virtual: boolean;
+    landing_company_shortcode: 'svg' | 'costarica' | 'maltainvest' | 'malta' | 'iom';
```

Comment:
We're only supporting `svg` and `maltainvest`.
Any reason to keep the rest of landing companies